packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -80,6 +113,38 @@ type Story @refersTo(name: \\"stories\\") @model {
 "
 `;
 
+exports[`generate graphqlSchemaFromSQLSchema creates graphql schema from "postgres" "newsNameChange" schema 1`] = `
+"input AMPLIFY {
+ engine: String = \\"postgres\\"  
+}
+
+
+type Article @refersTo(name: \\"articles\\") @model {
+  pubId: Int! @refersTo(name: \\"pub_id\\") @primaryKey(sortKeyFields: [\\"pubType\\"])
+  pubType: String! @refersTo(name: \\"pub_type\\") @default(value: \\"'A'::bpchar\\")
```

Comment:
The default values generated here don't look right. We would need to remove the `::<type>` part.

Reply:
Looks like this an existing bug. I'll fix this in a separate PR.